modules/backing-up-red-hat-quay-operator.adoc

```diff
@@ -308,7 +308,7 @@ spec:
 ----
 <1> Re-enables auto scaling of Quay, Clair and Mirroring workers again (if desired)
 <2> Replica overrides are removed again to scale the Quay components back up
-<3> Set to `false` on IBM Power and IBM Z deployments.
+<3> Set to `true` on IBM Power and IBM Z deployments.
```

Comment:
@alishaIBM this line should be removed. It does not make sense to set it to true always for P/Z since it is now a supported feature.

Reply:
Ack. Thanks.